src/config/extract-configuration-from-environment.js

```diff
@@ -54,6 +54,12 @@ const extractConfigurationFromEnvironmentVariable = function() {
     SENTRY_MAX_VALUE_LENGTH: 1000,
     REDIS_URL: process.env.REDIS_URL || 'redis://127.0.0.1:6379',
     NOTIFICATION_URLS,
+    S3_ACCESS_KEY: process.env.S3_ACCESS_KEY,
+    S3_SECRET_ACCESS_KEY: process.env.S3_SECRET_ACCESS_KEY,
+    S3_ENDPOINT: process.env.S3_ENDPOINT,
+    S3_REGION: process.env.S3_REGION,
+    S3_BUCKET_NAME: process.env.S3_BUCKET_NAME,
+    S3_OBJECT_NAME_PREFIX: process.env.S3_OBJECT_NAME_PREFIX,
```

Comment:
Grouper les variables, ce qui permet de supprimer le préfixe.
Les nommer en camelCase comme sur le monorepo
https://github.com/1024pix/pix/blob/dev/api/lib/config.js

```suggestion
    backupToS3: {
	    accessKey: process.env.S3_ACCESS_KEY,
	    secretAccessKey: process.env.S3_SECRET_ACCESS_KEY,
	    endpoint: process.env.S3_ENDPOINT,
	    regionName: process.env.S3_REGION,
	    bucketName: process.env.S3_BUCKET_NAME,
	    objectNamePrefix: process.env.S3_OBJECT_NAME_PREFIX
    },
```